src/frameworks/openal.rs

```diff
@@ -291,6 +291,15 @@ fn alBufferDataStatic(
     alBufferData(env, buffer, format, data, size, samplerate);
 }
 
+// Apple-specific extension to OpenAL
+fn alcMacOSXMixerOutputRate(_env: &mut Environment, value: ALdouble) {
```

Comment:
Ooh. For Super Monkey Ball I was able to get away with returning NULL when it tried to look up the address of this function, but I guess some apps try to call it directly. This is an Apple-specific extension that OpenAL Soft doesn't have, but I assume it's unimportant, so we can probably just make it do nothing, and remove the Super Monkey Ball hack.

Reply:
Yeah. This appears to be used to set the output sample rate. There's a book (from 2010!) which goes into more details about it here:

https://www.google.com/books/edition/Beginning_iPhone_Games_Development/xtrEvAdSOWEC?hl=en&gbpv=1&dq=alcMacOSXMixerOutputRate&pg=PA388&printsec=frontcover

Reply:
Sadly I can't see that, maybe because of my region. I guess it's nothing important though.

Reply:
Yeah. The book just talks about how setting it to the same sample rate as your audio files can possibly increase performance by avoiding the need for sample rate conversions.